api/src/main/java/org/openmrs/module/appointments/notification/impl/DefaultMailSender.java

```diff
@@ -88,12 +94,12 @@ private Session getSession() {
                         log.info("Could not load mail properties from application data directory. Loading from OMRS settings.");
                         sessionProperties = mailSessionPropertiesFromOMRS();
                     }
-                    final String user = sessionProperties.getProperty("mail.user");
-                    final String password = sessionProperties.getProperty("mail.password");
-                    if (StringUtils.isNotBlank(user) && StringUtils.isNotBlank(password)) {
+                    final String username = sessionProperties.getProperty("smtp.username");
```

Comment:
any reason for changing the property names? the reason for such keys were to keep in sync with the OMRS global properties for mail config.

Reply:
maybe the property name should be mail.smtp.user, and mail.smtp.password .. if you are changing here, make sure the same is done in the mapping in method - DefaultMailSender.mailSessionPropertiesFromOMRS()

Reply:
@angshu, kept properties as mail.smtp.user and mail.smtp.password